synthtool/gcp/templates/python_library/docs/multiprocessing.rst

```diff
@@ -1,7 +1,7 @@
 .. note::
 
-   Because this client uses :mod:`grpcio` library, it is safe to
+   Because this client uses :mod:`grpc` library, it is safe to
```

Comment:
The package is named `grpcio` on PyPI https://pypi.org/project/grpcio/. I'm not sure if it's worth noting that separately or if the link being active is enough.

Reply:
I agree the naming is confusing but after visiting the link and following the [quickstart](https://grpc.io/docs/languages/python/quickstart/#grpc) it was clear to me that I should install the package `grpcio`.